src/main/java/com/github/dockerjava/netty/exec/BuildImageCmdExec.java

```diff
@@ -55,7 +55,13 @@ protected Void execute0(BuildImageCmd command, ResultCallback<BuildResponseItem>
         if (command.getTags() != null && !command.getTags().isEmpty()) {
             webTarget = webTarget.queryParamsSet("t", command.getTags());
         } else if (isNotBlank(command.getTag())) {
-            webTarget = webTarget.queryParam("t", command.getTags());
+            webTarget = webTarget.queryParam("t", command.getTag());
```

Comment:
there was a typo there, its supposed to be getTag(),  So i thought I should fix it. getTags() is if there are not multiple tags and this elseIf loop should execute getTag(). The same reference is found in jaxrs/BuildImageCmdExec.java file too.